string rosbag
string status
uint64 bytes_total_size
uint64 bytes_transferred
